gcloud/_helpers.py

```diff
@@ -37,6 +37,7 @@
 
 _NOW = datetime.datetime.utcnow  # To be replaced by tests.
 _RFC3339_MICROS = '%Y-%m-%dT%H:%M:%S.%fZ'
+_RFC3339_NO_FRACTION_NO_ZULU = '%Y-%m-%dT%H:%M:%S'
```

Comment:
Zulu from NATO phonetic alphabet? Seems like the name might be confusing to some.